kimchi/src/circuits/gate.rs

```diff
@@ -186,18 +182,35 @@ pub enum LookupsUsed {
     Joint,
 }
 
+/// Enumerates the different 'fixed' lookup tables used by individual gates
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
+pub enum GateLookupTable {
+    Xor,
+}
+
+pub fn get_table<F: FftField>(table_name: GateLookupTable) -> Vec<Vec<F>> {
```

Comment:
Missing doc on this and the previous enum

Reply:
The name already says what it does; I don't know what comment I could add that wouldn't be redundant.

Reply:
as above, public functions need to be documented, even if redundant